Rise-Vision/common-template
# Retrieve logo and colors from display.json company branding object

## Description
Allow component to register for Branding logoFile updates
Update html page style sheets with the company colours via predefined classes

## Motivation and Context
Add support for Company Branding as per the Branding epic.

## How Has This Been Tested?
The changes have been tested locally via the Chrome Player with the demo page
Tested demo via Editor Preview

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
**Manual Test**  As per the above
**Automated Test** Unit tests were added for the change
**Monitoring** Will test the Beta release before releasing to Production. Once in Stable will monitor that no errors are occurring
**Rollback** Rollback can be done by reverting the PR and following the deployment process
**Documentation** Documentation updated as per the design doc for the Epic: https://docs.google.com/document/d/1XZUcQ6D9oWjMxrllbZKZLCDV3D8W89OyqrUMwVjEp_8/edit#
**Internal Communication** Will notify interested parties once deployed

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a
